contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -272,7 +272,35 @@ NXmpes(NXobject):
               exists: recommended
             identifier:
               exists: recommended
-          (NXdata):
+          raw_data(NXdata):
+            # ToDo:
+            # - With base class inheritance this should be
+            #   a field of type NXdata_mpes_detector
+            # - The field name should be aligned with the `data` field in 
+            #   the `NXdetector` base class. But some refinement is necessary there.
+            doc: |
+              Contains the raw data collected by the detector before calibration.
+              The data which is considered raw might change from experiment to experiment
+              due to hardware pre-processing of the data.
+              This field ideally collects the data with the lowest level of processing
+              possible.
+
+              The naming of fields should follow a convention to ensure compatibility.
+              It is recommend to use the following field names:
+
+              - **pixel_x**: Detector pixel in x direction.
+              - **pixel_y**: Detector pixel in y direction.
+              - **raw_time_of_flight**: The raw time of flight in DAQ clock pulses. Unit category: NX_PULSES
+              - **time_of_flight**: Total time of flight. Unit category: NX_TIME_OF_FLIGHT
+              - **time_of_flight_adc**: Time-of-flight values, analog-to-digital converted.
+              - **polar_angle_manipulator**: Polar rotation angle of the manipulator. Unit category: NX_ANGLE
+              - **azimuthal_angle_manipulator**: Azimuthal rotation angle of the manipulator. Unit category: NX_ANGLE
+              - **delay_position**: Delay position of delay line detector.  Unit category: NX_LENGTH
+              - **delay_position_ADC**: Delay position of delay line detector, analog-to-digital converted.
+              - **time_delay_x**: Time delay of delay line detector in x direction. Unit category: NX_TIME
+              - **time_delay_y**: Time delay of delay line detector in y direction. Unit category: NX_TIME
+              - **voltage**: Voltage applied to sample and sample holder. Unit category: NX_VOLTAGE
+              - **external_AXIS**: Describes an axis which is coming from outside the detectors scope.
```

Comment:
I guess the plan was to store uncalibrated data here, right? Then the language from `NXmpes/data` about calibrated axes doesn't make too much sense. My proposal:
```yaml
- **pixel_x**: Detector pixel in x direction.    # This was suggested in the first workshop IIRC, so it would keep it.
- **pixel_y**: Detector pixel in y direction.
- **kx**: (Un)calibrated x axis in k-space. Unit category: NX_ANY (e.g., 1/Angström).
- **ky**: (Un)calibrated y axis in k-space. Unit category: NX_ANY (1/Angström).
- **kz**: (Un)calibrated z axis in k-space. Unit category: NX_ANY (1/Angström).
- **angular0**: Fast-axis angular coordinate (or second slow axis if angularly integrated).
  Unit category: NX_ANGLE
- **angular1**: Slow-axis angular coordinate (or second fast axis if simultaneously dispersed in 2 dimensions)
  Unit category: NX_ANGLE
- **spatial0**: Fast-axis spatial coordinate (or second slow axis if spatially integrated)
  Unit category: NX_LENGTH
- **spatial1**: Slow-axis spatial coordinate (or second fast axis if simultaneously dispersed in 2 dimensions)
  Unit category: NX_LENGTH
- **delay**: (Un)calibrated delay time. Unit category: NX_TIME (s).
- **polarization_angle**: Linear polarization angle of the incoming or
  outgoing beam.
  Unit category: NX_ANGLE (° or rad)
- **ellipticity**: Ellipticity of the incoming or outgoing beam.
  Unit category: NX_ANGLE (° or rad)              
- **time_of_flight**: Total time of flight. Unit category: NX_TIME_OF_FLIGHT
- **time_of_flight_adc**: Time-of-flight values, analog-to-digital converted.
- **external_AXIS**: Describes an axis which is coming from outside the detectors scope.
```

Reply:
looks good to me.

Reply:
I have added here the (un)calibrated energy axis.